spec/matchers/have_run.rb

```diff
@@ -0,0 +1,25 @@
+RSpec::Matchers.define :have_run do |job_name|
+  match do |clock_test|
```

Comment:
Will `clock_test` only resolve appropriately if in the test, we have all uses of this in a `describe Clockwork::Test {}` block? 

If that's the case, I'm not sure if that's a valid assumption to make for users of the gem, or a reasonable assertion to enforce to use the matcher.

Please let me know if I'm understanding this correctly.